website/content/en/docs/concepts/disruption.md

```diff
@@ -291,16 +291,17 @@ metadata:
 
 #### Example: Disable Disruption on a NodePool
 
-NodePool `.spec.annotations` allow you to set annotations that will be applied to all nodes launched by this NodePool. By setting the annotation `karpenter.sh/do-not-disrupt: "true"` on the NodePool, you will selectively prevent all nodes launched by this NodePool from being considered in disruption actions.
+NodePool `.spec.disruption.budgets` allows you to set disruption budgets for nodes launched by this NodePool, preventing them from being considered in disruption actions.
```

Comment:
Wording suggestion, flows a little better IMO and specifies that this only blocks voluntary disruption. 
```suggestion
To disable disruption for all nodes launched by a NodePool, you can configure its `.spec.disruption.budgets`. Setting a budget of zero nodes will prevent any of those nodes from being considered for voluntary disruption.
```

Reply:
Agreed. Made an update.